pkg/overlay/cache_test.go

```diff
@@ -94,6 +95,46 @@ func testCache(ctx context.Context, t *testing.T, store overlay.DB) {
 	}
 }
 
+func TestUpdateContained(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+
+		cache := planet.Satellites[0].DB.OverlayCache()
+
+		nodeID0 := planet.StorageNodes[0].ID()
+		nodeID1 := planet.StorageNodes[1].ID()
```

Comment:
Could you move this declaration down, close to its first usage?

Reply:
sure!